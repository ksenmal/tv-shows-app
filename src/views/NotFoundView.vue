<script setup>
import { RouterLink } from "vue-router";
</script>

<template>
  <div class="page-not-found">
    <p>Page not found.</p>
    <p>Go to <RouterLink to="/">Main page</RouterLink></p>
  </div>
</template>

<style scoped>
.page-not-found {
  margin-top: 2rem;
  text-align: center;
  font-size: 1.5rem;
}
</style>
