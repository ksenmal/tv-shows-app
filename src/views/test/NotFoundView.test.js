import NotFoundView from "../NotFoundView.vue";
import { shallowMount } from "@vue/test-utils";
import { describe, expect, test } from "vitest";

describe("NotFoundView suite", () => {
  test("NotFoundView snapshot", () => {
    const wrapper = shallowMount(NotFoundView);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
