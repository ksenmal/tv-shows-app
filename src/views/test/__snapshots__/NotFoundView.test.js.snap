// Vitest Snapshot v1

exports[`NotFoundView suite > NotFoundView snapshot 1`] = `
"<div class=\\"page-not-found\\" data-v-071430af=\\"\\">
  <p data-v-071430af=\\"\\">Page not found.</p>
  <p data-v-071430af=\\"\\">Go to <router-link-stub to=\\"/\\" replace=\\"false\\" custom=\\"false\\" ariacurrentvalue=\\"page\\" data-v-071430af=\\"\\"></router-link-stub>
  </p>
</div>"
`;
